acceptance/bundle/resources/clusters/deploy/update-and-resize-autoscale/databricks.yml.tmpl

```diff
@@ -0,0 +1,22 @@
+bundle:
+  name: test-deploy-cluster-autoscale
+
+workspace:
+  root_path: ~/.bundle/$UNIQUE_NAME
+
+resources:
+  clusters:
+    test_cluster:
+      cluster_name: test-cluster-$UNIQUE_NAME
+      spark_version: $DEFAULT_SPARK_VERSION
+      node_type_id: $NODE_TYPE_ID
+      num_workers: 2
+
+  jobs:
+    foo:
+      name: test-job-with-cluster-$UNIQUE_NAME
+      tasks:
+        - task_key: my_spark_python_task
+          existing_cluster_id: "${resources.clusters.test_cluster.cluster_id}"
```

Comment:
Question - Is .cluster_id intentional instead of .id?

Reply:
Yes, cluster_id is the field that is returned from the API, and we do the same in the rest of the codebase. But .id should work as well, and we have tests which use it as well